examples/all-clusters-app/esp32/main/DeviceCallbacks.cpp

```diff
@@ -254,6 +258,26 @@ void DeviceCallbacks::OnColorControlAttributeChangeCallback(EndpointId endpointI
 }
 #endif
 
+void DeviceCallbacks::OnIdentifyPostAttributeChangeCallback(EndpointId endpointId, AttributeId attributeId, uint8_t * value)
+{
+    if (attributeId == ZCL_IDENTIFY_TIME_ATTRIBUTE_ID)
+    {
+        uint8_t identifyTime = *value;
```

Comment:
This does not look right to me.  IdentifyTime is of type `int16u`, no?

`PostAttributeChangeCallback` was passed the size of the data to read from `value`.  We're ignoring that size, and reading the wrong amount of data....

Instead we should be passing the size in here, checking that it matches expectations (i.e. is 2), and if so reading the value as a `uint16_t`, no?

Reply:
added this change